game/views.py

```
@@ -3889,3 +3892,112 @@ def forum_reply_delete(request, reply_id):
+    if request.method == "POST":
+        form = AvatarUploadForm(request.POST, request.FILES)
+        if form.is_valid():
+            uploaded_file = form.cleaned_data["avatar"]
+            try:
+                img = Image.open(uploaded_file)
+                # Eagerly decode the entire image to catch truncated /
+                # corrupt files before any transformation takes place.
+                try:
+                    img.load()
+                except Exception as exc:
+                    raise ValueError("Image file appears to be corrupt or truncated.") from exc
```

Ruff (0.15.18)

[warning] Avoid specifying long messages outside the exception class

(TRY003)